backend/api/EventHandlers/MqttEventHandler.cs

```diff
@@ -150,26 +152,6 @@ private async void OnIsarStatus(object? sender, MqttReceivedArgs mqttArgs)
 
             if (robot.IsRobotReadyToStartMissions())
             {
-                try
```

Comment:
Removing this as we are just removing the currentMissionId if receiving a status indicating that there is no ongoing mission, without actually handling the potentially ongoing mission